book/from_flat_php_to_symfony2.rst

```diff
@@ -705,7 +737,7 @@ for example, the list template written in Twig:
         </ul>
     {% endblock %}
 
-The corresponding ``layout.html.twig`` template is also easier to write:
+The corresponding ``layout.html.php`` template is also easier to write in Twig:
```

Comment:
To be honest, I don't fully get your changes here. Line 718 now says "lets rewrite ``layout.html.php`` template in Twig:" and directly after it, you have this line also mentioning "The corresponding ``layout.html.php`` template is also easier to write in Twig:"